DEPRECATION.md

```diff
@@ -169,13 +169,21 @@ const { BaseQuery } = require('@cubejs-backend/schema-compiler');
 
 **Deprecated in Release: v0.26.0**
 
-Previously our authentication mechanism was tightly bound to Express. Due to the
-[deprecation of the Express-based API](#embedding-cubejs-within-express), we
-want our authentication API to be generic and not transport-specific. We now
-recommend using [`checkAuth`][ref-checkauth] as a transport-agnostic method of
+> 
+
+Motivation:
+
+1. The `checkAuthMiddleware` was tightly bound to Express, [which was deprecated](#embedding-cubejs-within-express).
+2. HTTP is not only the transport for Cube.js, We support WebSocket server, which can be a default way in the feature. In this case
+   we want our authentication API to be not transport-specific.
```

Comment:
```suggestion
The `checkAuthMiddleware` was tightly bound to Express, [which has been deprecated](#embedding-cubejs-within-express). Since Cube.js supports HTTP **and** WebSockets as transports, we want our authentication API to not rely on transport-specific details.
```